localisation/english/btr_traits_l_english.yml

```diff
@@ -47,4 +48,14 @@
  btr_human_rugged_ind_desc:0 "Just as The Alliance asserts itself in the face of the galactic community, many humans aim to assert their own desires to the Alliance. Many humans prefer to go their own way, some seeking a home in the lawless terminus systems, and some content to live quietly on the fringes of the Alliance and ignore it."
  btr_leader_trait_twice_bright: "Twice as Bright"
  btr_leader_trait_twice_bright_desc: "Humans have an insatiable desire to achieve results immediately, rather than expecting change to take generations. Some aliens see this as a lack of focus, or intransigence. Nonetheless, Humans are driven to strive, suffer, and overcome by the knowledge of their mortality to a degree not seen in their peers."
- 
+
+### Yahg Traits ###
+
+btr_yahg_trait_core:0 "Yahg"
+btr_yahg_trait_core_desc:0 "Consummate predators, the Yahg are a sentient race of towering humanoids native to the world of Parnack, known for their violent and aggressive nature." 
+btr_yahg_trait_apex_pred:0 "Apex Predator"
+btr_yahg_trait_apex_pred_desc:0 "Stronger and more cunning than almost any alien species, the Yahg evolved to fill the niche of apex predator on Parnack. On the other hand, being exclusively carnivores, they have a very slow demographic growth rate."
+btr_yahg_trait_truthsay:0 "Truthsayers"
+btr_yahg_trait_truthsay_desc:0 "A keen sensitivity to movement and light allows the Yahg to easily read the body language of any species, making it almost impossible to lie to a Yahg. This also makes them excel at intelligence agents."
```

Comment:
This also makes them excel as intelligence agents

Reply:
Fixed